components/Login.tsx

```diff
@@ -1,26 +1,57 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import {
   ImageBackground,
   Text,
   TouchableOpacity,
   View,
   TextInput,
+  Alert,
 } from 'react-native';
 import * as WebBrowser from 'expo-web-browser';
 import { imageStyles, styles, loginStyles, touchableStyles } from '../styles';
 import { useAuthContext } from '../contexts';
+import { useUserContext } from '../contexts';
+import axios from 'axios';
 
 WebBrowser.maybeCompleteAuthSession();
 
 function Login() {
-  const { login, response, promptAsync } = useAuthContext();
+  const { setUserDetails } = useUserContext();
+  const { googleLogin, response, promptAsync } = useAuthContext();
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
 
   useEffect(() => {
     if (response) {
-      login();
+      googleLogin();
     }
   }, [response]);
 
+  function noGoogleLogin() {
+    axios
+      .post('https://api-wanderlust-dogs.herokuapp.com/users/login/mobile', {
+        email,
+        mobilePassword: password,
+      })
+      .then(response => {
+        const id = response.data.id;
+        axios
+          .get(`https://api-wanderlust-dogs.herokuapp.com/users/${id}`)
+          .then(response => {
+            const { email, username, Trips, id } = response.data;
+            setUserDetails({
+              email,
+              id,
+              username,
+              Trips,
+              accessToken: null,
+            });
+          })
+          .catch(error => Alert.alert('User info fetch failed'));
+      })
+      .catch(error => Alert.alert('Login failed, try again'));
```

Comment:
Alert.alert takes 2 arguments: a title and a body. We should use both for these alerts so it doesn't look weird and empty.

Reply:
mainly had them for debugging inside the sea of console logs (very big objects come from db calls)

Reply:
made errors more readable